blocks/section-header/section-header.css

```diff
@@ -0,0 +1,42 @@
+.section-header {
+  grid-row-gap: 0.75rem;
+
+  @media (min-width: 80rem) {
+    grid-row-gap: 1rem;
+  }
+}
+
+main > .section > .grid-container {
+  /* remove default margin-bottom as section headings sit close to content */
+  margin-block-end: 2rem;
+
+  @media (min-width: 80rem) {
+    margin-block-end: 2.5rem;
+  }
+}
+
+.section-header__description {
+  margin: 0;
+  margin-block-start: 0;
+}
+
+.section-header__button--hide-at-small {
+  /* see media query below for display change */
+  display: none;
+}
```

Comment:
Opted to nest this in the mq since I was already doing that with layout changes, instead of nesting the mq in this rule, but can change if you have a preference

Reply:
I'd rather nest the MQs within the relevant selectors - feels like a dev experience benefit to me, reduce the amount of scrolling back and forth if there is a need to modify all related styles.